### DISCUSSION TITLE
Getting Started with Tekton - Fundamental Concepts

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#74] [BlogPosts]
# Getting Started with Tekton - Fundamental Concepts

Tekton provides a cloud-native, standardized set of building blocks for CI/CD systems. It runs natively on Kubernetes and can target any platform, language, or cloud. It extends the Kubernetes API and provides custom resources to create CI/CD pipelines.

https://pradeepl.com/blog/kubernetes/tekton-getting-started-fundamental-concepts/

**Comment by kubectl9** [comment]
Hi Pradeep, It's a great article. I have docker desktop installed (and Kubernetes enabled) on my Windows machine. Can you provide a simple example of building a simple Tekton pipeline? Thanks.

**Reply by PradeepLoganathan** [reply]
Hello @kubectl9 You would need to install the prerequisites for tekton on the kubernets cluster you create locally. You can simply apply the helloworld task defined in the blog post. You can then apply the hello-world-pipeline which uses the helloworld task . You can then apply the run-hello-pipeline pipelinerun which will trigger the pipeline containing the task. Hope this helps,
